cmd/ocm-backplane/cloud/ssm.go

```diff
@@ -126,12 +128,27 @@ func getInstanceID(nodeName string, config *rest.Config) (string, error) {
 }
 
 func startSSMsession(cmd *cobra.Command, argv []string) error {
-	if ssmArgs.node == "" {
-		return fmt.Errorf("--node flag is required")
+	// Define the proxy URL explicitly
+	proxyURL := "http://squid.corp.redhat.com:3128"
```

Comment:
How about loading from config file like this?
```
import (
	"encoding/json"
	"os"
)

type Config struct {
	ProxyURL string `json:"proxy_url"`
}

func loadConfig(path string) (Config, error) {
	file, err := os.Open(path)
	if err != nil {
		return Config{}, err
	}
	defer file.Close()

	var config Config
	err = json.NewDecoder(file).Decode(&config)
	return config, err
}

// Usage
config, err := loadConfig("config.json")
if err != nil {
	log.Fatalf("Error loading config: %v", err)
}
proxyURL := config.ProxyURL
```

Reply:
Updating : We don't need to define a new struct for proxy.

Reply:
IMO, We can use the BackplaneConfiguration struct and the GetBackplaneConfiguration function to dynamically read and update the proxy URL in your backplane-cli